Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -15,21 +16,106 @@ public Inventory()
 
         public Item[] GetItems(string name)
         {
-            return new Item[0];
+            var foundItems = new Item[0];
+            foreach (var item in _items)
+            {
+                if (item.GetName() == name)
+                {
+                    foundItems = Add(item);
+                }
+            }
+
+            return foundItems;
         }
 
         public void AddItem(Item item)
         {
+            _items = Add(item);
+        }
 
+        public Item[] Add(Item item)
+        {
+            var newArray = new Item[_items.Length + 1];
+            for (int i = 0; i <= _items.Length - 1; i++)
+            {
+                newArray[i] = _items[i];
+            }
+            newArray[^1] = item;
+            return newArray;
         }
 
+        public void RemoveByName(string name)
+        {
+            var newArray = new Item[_items.Length - 1];
+            if (_items == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'false'; some subsequent code is never executed.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5138976)